arduino-sketch/bleAndMatrix.ino

```diff
@@ -0,0 +1,86 @@
+// Configuration for the Arduino R4 WiFi
+
+#include <ArduinoBLE.h>
+#include "Arduino_LED_Matrix.h"
+
+BLEService customService("ee4e5719-e214-4217-ba3e-c5de249a3b3c");                                            // Custom service UUID
```

Comment:
I assume this is the hashed data connection for bluetooth?

Reply:
That is something called [uuid](https://microchipdeveloper.com/xwiki/bin/view/applications/ble/introduction/bluetooth-architecture/bluetooth-host-layer/bluetooth-generic-attribute-profile-gatt/UUIDs/). I suggest you don't dive too much in it, I spent quite some time